.travis.yml

```diff
@@ -8,6 +8,6 @@ branches:
     - master
 
 script:
-  - GOTEST_FLAGS="-p 2 -parallel 2" make test
+  - GOTEST_FLAGS="-p 3 -parallel 1" make test
```

Comment:
@pierresouchay since we are trying to fix this in another PR can you remove it here? Even if CI fails on the PR due to flakiness we can merge anyway if we check the failures are clearly unrelated to this and random.

Reply:
Yes, I am gonna try to remove it, but I have the feeling that using parallel 1 really helps (while -p is to avoid having full timeout of 10m, but -p2 should be good)